ue4docker/dockerfiles/diagnostics/20gig/windows/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# escape=`
+ARG BASETAG
+FROM mcr.microsoft.com/windows/servercore:${BASETAG} as intermediate
+
+# Add a sentinel label so we can easily identify intermediate images
+LABEL com.adamrehn.ue4-docker.sentinel="1"
+
+# Create a file slightly over 20GiB
+RUN fsutil file createnew C:\bigfile.txt 22548578304
```

Comment:
This step will fail if the 8gB bug is present. Should we try to avoid that? Is it possible/reasonable to use one diagnostic to test for both issues, and distinguish the result?

Reply:
Oh, that's why you were using 3x7GiB. I dunno whether we want to distinguish them. What we actually want in the end is to be able to copy > 20GiB no matter how it is organized in files. I can easily switch to 3x7GiB is you think this is preferred.

Reply:
Okay, I switched it to 3x7GiB.